### DISCUSSION TITLE
I was approved for the free GitHub Education benefits, but I haven't received my benefits yet.

### DISCUSSION THREAD

**Discussion by sundoopark** [community/community#147583] [GitHub Education]
### Hi everyone,

I was approved for the free GitHub Education benefits, but I haven't received my benefits yet.
When should I expect the free GitHub Education benefits to be available?

**Comment by davevad93** [comment]
Hi @sundoopark ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @sundoopark was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by sundoopark** [comment]
Thanks for your response. I just got a notification that my benefits are now available.
However, when I go to the Github Copilot tab, I see the below message:

```
You are using Copilot for free
Responses are limited to 2,000 code completions and 50 chat messages per month.
```

Is this normal or will it change to Copilot Pro eventually?

**Reply by davevad93** [reply]
Hey @sundoopark you're welcome. The message on your `Copilot` tab means that you are still on the free tier, to redeem your `PRO` offer go to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select [Copilot](https://education.github.com/pack/redeem/copilot-student). If after doing this you are still unable to redeem your offer, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Comment by sundoopark** [comment]
Hi @davevad93 , I could see Copilot under "All offers" section, but there is no way to select or enable it.

<img width="1451" alt="image" src="https://github.com/user-attachments/assets/1094659a-2570-487d-b6dd-e56797b7ff2b" />
Do you think I need to submit a ticket to the link you provided? BTW, I'm a faculty, not a student.

**Reply by davevad93** [reply]
Hey @sundoopark , yes, raise a ticket [here](https://support.github.com/contact/education) and select `I am a verified student having trouble redeeming my Global Campus offers` (I know you're a faculty but that's the only option available).
